apps/web/pages/apps/[slug]/index.tsx

```diff
@@ -108,9 +109,11 @@ export const getStaticProps = async (ctx: GetStaticPropsContext) => {
   const { content, data } = sourceSchema.parse({ content: result.content, data: result.data });
   if (data.items) {
     data.items = data.items.map((item) => {
-      if (typeof item === "string" && !item.includes("/api/app-store")) {
-        // Make relative paths absolute
-        return `/api/app-store/${appDirname}/${item}`;
+      if (typeof item === "string") {
```

Comment:
Keep the logic to convert the asset into URL in 2 places only.
1. For DESCRIPTION.md items
2. For app.logo